charts/base-cluster/README.md.gotmpl

```diff
@@ -37,24 +37,153 @@ git push
+
+### Component [ingress](#ingress)
+
+The included [`nginx` ingress-controller](https://docs.nginx.com/nginx-ingress-controller)
+only works for the `IngressClassName: nginx`.
+
+#### TLS
+
+1. add `kubernetes.io/tls-acme: "true"` to your Ingress's annotations
+    - additionally, although not advised unless you know what you're doing,
+      you can explicitly choose the Issuer by using these annotations:
+      - `cert-manager.io/cluster-issuer: letsencrypt-staging`
+      - `cert-manager.io/cluster-issuer: letsencrypt-production`
+
+#### IP Address
+
+If you want to make sure that, in the event of a catastrophic failure, you keep the
+same IP adress, you should roll this out, get the assigned IP
+(`kubectl -n ingress-nginx get svc ingress-nginx-controller -o jsonpath='{.status.loadBalancer.ingress}'`)
+and set `.ingress.IP=<ip>`. This makes sure the IP is kept in your project (may incur cost!)
+which means you can re-use it later or after recovery.
+
+### Component [flux](#flux)
+
+[Flux](https://fluxcd.io) is used to deploy resources to your cluster and to
+keep them in sync.
+
+Flux can also auto-update images and HelmReleases.
+
+You can create any number of gitRepository connections, with SSH(recommended)
+or https checkout, with or without SOPS, ... .
+
+### Component [kyverno](#kyverno)
+
+You can optionally enable [kyverno](https://kyverno.io), which is a policy
+system, allowing you to specify in-depth policies to prevent or force certain
+things in your cluster.
+
+### Component [monitoring](#monitoring)
+
+#### Sub-Component [prometheus](#monitoring_prometheus)
+
+[Prometheus](https://prometheus.io) takes care of scraping metrics and alerting.
+
+#### Sub-Component [grafana](#monitoring_grafana)
+
+[Grafana](https://grafana.com) is used to create dashboards to visualize your
+metrics and the health of your cluster and applications.
+
+#### Sub-Component [loki](#monitoring_loki)
+
+[Loki](https://grafana.com/oss/loki) collects logs from across the cluster and
+allows to have a centralized, non-CLI, view of the logs and to create alerting
+based on them.
+
+#### Sub-Component [metrics-server](#monitoring_metricsServer)
+
+[Metrics Server](https://github.com/kubernetes-sigs/metrics-server) implements
+the [kubernetes Metrics API](https://github.com/kubernetes/metrics) to allow
+for [Horizontal Autoscaling](https://kubernetes.io/docs/tasks/run-application/horizontal-pod-autoscale),
+[Vertical Autoscaling](https://github.com/kubernetes/autoscaler/tree/master/vertical-pod-autoscaler)
+and to allow `kubectl top` and tools like [k9s](https://k9scli.io) to show
+resource usage for your pods and nodes.
+
+#### Sub-Component [securityScanning](#monitoring_securityScanning)
+
+The included [trivy](https://aquasecurity.github.io/trivy-operator) scans the
+running workload in your cluster for CVEs and creates
+[Custom Resources](https://aquasecurity.github.io/trivy-operator/v0.12.1/docs/crds).
```

Comment:
Maybe add a sentence that you can also monitor the found CVEs in your cluster.
Somethin like:

```
... creates Custom Resources to monitor the results.
```
or something like that.